gpflow/optimizers/scipy.py

```diff
@@ -109,7 +123,10 @@ def eval_func(
         variables: Sequence[tf.Variable],
         compile: bool = True,
         allow_unused_variables: bool = False,
+        tffun_args: Optional[Mapping[str, Any]] = None,
```

Comment:
I am thinking of making this a non-optional arg. This will remove the 2 lines below and improve the code coverage. This method is not directly tested and since I believe this is an internal method, it is okay to change its interface. Is that okay?

Reply:
I have gone ahead and made this change, as otherwise the codecov check was failing.

Reply:
:+1: